### PR TITLE
Fixes to udemy API, fixes: #1 and update to Kafka 1.1.1 / Cofluent 5.0.1

### PR DESCRIPTION
**Pull request by matiwinnetou**
It turns out udemy API removed @default, so now one has to enumerate all fields one wants. This fixes test error issue found in: #1

### CODE REVIEW

**Comment by matiwinnetou** [comment]
THIS IS WORK in progres, for now only tests pass and udemy reviews producer works.

File: udemy-reviews-producer/src/main/java/com/github/simplesteph/kafka/producer/udemy/client/UdemyRESTClient.java [hunk 1]
```diff
@@ -83,37 +82,31 @@ public ReviewApiResponse reviewApi(Integer pageSize, Integer page) throws HttpEx
...
         Review.Builder reviewBuilder = Review.newBuilder();
-        reviewBuilder.setContent(reviewJson.getString("content"));
         reviewBuilder.setId(reviewJson.getLong("id"));
-        reviewBuilder.setRating(reviewJson.getBigDecimal("rating").toPlainString());
         reviewBuilder.setTitle(reviewJson.getString("content"));
+        reviewBuilder.setContent(reviewJson.getString("content"));
+        reviewBuilder.setRating(reviewJson.getBigDecimal("rating").toPlainString());
         reviewBuilder.setCreated(DateTime.parse(reviewJson.getString("created")));
         reviewBuilder.setModified(DateTime.parse(reviewJson.getString("modified")));
+        reviewBuilder.setUserModified(DateTime.parse(reviewJson.getString("user_modified")));
```
Comment (matiwinnetou) [inline udemy-reviews-producer/src/main/java/com/github/simplesteph/kafka/producer/udemy/client/UdemyRESTClient.java]: they added user_modified

File: udemy-reviews-producer/src/main/java/com/github/simplesteph/kafka/producer/udemy/client/UdemyRESTClient.java [hunk 2]
```diff
@@ -83,37 +82,31 @@ public ReviewApiResponse reviewApi(Integer pageSize, Integer page) throws HttpEx
...
         reviewBuilder.setUser(jsonToUser(reviewJson.getJSONObject("user")));
-        reviewBuilder.setCourse(jsonToCourse(reviewJson.getJSONObject("course")));
```
Comment (matiwinnetou) [inline udemy-reviews-producer/src/main/java/com/github/simplesteph/kafka/producer/udemy/client/UdemyRESTClient.java]: They removed course json node

File: udemy-reviews-producer/src/main/resources/avro/udemy-review.avsc [hunk 3]
```diff
@@ -3,21 +3,11 @@
   "type": "record",
   "name": "User",
   "fields": [
-    {"name": "id", "type": "long", "doc": "User ID in Udemy's DB"},
     {"name": "title",  "type": "string"},
     {"name": "name",  "type": "string", "doc": "first name"},
     {"name": "display_name",  "type": "string"}
   ]
 },
-{"namespace": "com.github.simplesteph.avro.udemy",
```
Comment (matiwinnetou) [inline udemy-reviews-producer/src/main/resources/avro/udemy-review.avsc]: course node is no longer part of the review

**Comment by matiwinnetou** [comment]
IGNORE THIS FOR NOW, I need to learn UDEMY API...

File: udemy-reviews-producer/src/main/resources/avro/udemy-review.avsc [hunk 4]
```diff
@@ -3,7 +3,6 @@
   "type": "record",
   "name": "User",
   "fields": [
-    {"name": "id", "type": "long", "doc": "User ID in Udemy's DB"},
```
Comment (matiwinnetou) [inline udemy-reviews-producer/src/main/resources/avro/udemy-review.avsc:6]: user node no longer reveals their ID. Perhaps it was a security issue, who knows.

File: udemy-reviews-aggregator/src/main/java/com/github/simplesteph/kafka/streams/udemy/aggregator/AggregatorMain.java [hunk 5]
```diff
@@ -42,40 +49,44 @@ private void start() {
 
     private Properties getKafkaStreamsConfig() {
         Properties config = new Properties();
-        config.put(StreamsConfig.APPLICATION_ID_CONFIG, appConfig.getApplicationId());
-        config.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, appConfig.getBootstrapServers());
-        config.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+        config.put(APPLICATION_ID_CONFIG, appConfig.getApplicationId());
```
Comment (simplesteph) [inline udemy-reviews-aggregator/src/main/java/com/github/simplesteph/kafka/streams/udemy/aggregator/AggregatorMain.java]: Can we please revert those?

**Review by simplesteph — CHANGES_REQUESTED** [review]
thanks for this. Just some minor changes to revert and I'll merge!

File: udemy-reviews-aggregator/src/main/java/com/github/simplesteph/kafka/streams/udemy/aggregator/AggregatorMain.java (hunk 5 above)
Comment (matiwinnetou) [reply udemy-reviews-aggregator/src/main/java/com/github/simplesteph/kafka/streams/udemy/aggregator/AggregatorMain.java]: Sure I can

Comment (matiwinnetou) [reply udemy-reviews-aggregator/src/main/java/com/github/simplesteph/kafka/streams/udemy/aggregator/AggregatorMain.java]: Yes I will revert those code style changes today.

**Review by simplesteph — APPROVED** [review]

**Comment by simplesteph** [comment]
thanks @matiwinnetou for the contribution! Looks great :)

**Pull request merged**